pkg/scheme/scheme.go

```diff
@@ -0,0 +1,36 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scheme
+
+import (
+	"github.com/istio-ecosystem/sail-operator/api/v1alpha1"
+	multusv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+
+	networkingv1alpha3 "istio.io/client-go/pkg/apis/networking/v1alpha3"
+)
+
+var Scheme = runtime.NewScheme()
+
+func init() {
+	utilruntime.Must(clientgoscheme.AddToScheme(Scheme))
+	utilruntime.Must(multusv1.AddToScheme(Scheme))
+	utilruntime.Must(networkingv1alpha3.AddToScheme(Scheme))
+
+	utilruntime.Must(v1alpha1.AddToScheme(Scheme))
+	// +kubebuilder:scaffold:scheme
```

Comment:
do we need this?

Reply:
Not sure. This was in the original file, but we don't really use kubebuilder to add new CRDs (at least I prefer to add them manually).